src/unity/python/README.md

```diff
@@ -1,132 +0,0 @@
-
-<p align="center"><img src="https://docs-assets.developer.apple.com/published/a2c37bce1f/689f61a6-1087-4112-99d9-bbfb326e3138.png" alt="Turi Create" width="600"></p>
-
-With Turi Create, you can can tackle a number of common scenarios:
-* [Recommender systems](https://apple.github.io/turicreate/docs/userguide/recommender/introduction.html)
-* [Image classification](https://apple.github.io/turicreate/docs/userguide/image_classifier/introduction.html)
-* [Image similarity](https://apple.github.io/turicreate/docs/userguide/image_similarity/introduction.html)
-* [Object detection](https://apple.github.io/turicreate/docs/userguide/object_detection/introduction.html)
-* [Activity classifier](https://apple.github.io/turicreate/docs/userguide/activity_classifier/introduction.html)
-* [Text classifier](https://apple.github.io/turicreate/docs/userguide/text_classifier/introduction.html)
-
-You can also work with essential machine learning models, organized into algorithm-based toolkits:
-* [Classifiers](https://apple.github.io/turicreate/docs/userguide/supervised-learning/classifier.html)
-* [Regression](https://apple.github.io/turicreate/docs/userguide/supervised-learning/regression.html)
-* [Graph analytics](https://apple.github.io/turicreate/docs/userguide/graph_analytics/intro.html)
-* [Clustering](https://apple.github.io/turicreate/docs/userguide/clustering/intro.html)
-* [Nearest Neighbors](https://apple.github.io/turicreate/docs/userguide/nearest_neighbors/nearest_neighbors.html)
-* [Topic models](https://apple.github.io/turicreate/docs/userguide/text/intro.html)
-
-System Requirements
--------------------
-
-* Python 2.7 (Python 3.5+ support coming soon)
-* x86\_64 architecture
-* macOS 10.11+, Linux with glibc 2.12+ (including WSL on Windows 10)
-
-Installation
-------------
-
-For detailed instructions for different varieties of Linux see [LINUX\_INSTALL.md](https://github.com/apple/turicreate/LINUX_INSTALL.md).
-For common installation issues see [INSTALL\_ISSUES.md](https://github.com/apple/turicreate/INSTALL_ISSUES.md).
-
-We recommend using virtualenv to use, install, or build Turi Create. 
-Be sure to install virtualenv using your system pip.
-
-```shell
-pip install virtualenv
-```
-
-The method for installing *Turi Create* follows the
-[standard python package installation steps](https://packaging.python.org/installing/).
-To create a Python virtual environment called `venv` follow these steps:
-
-```shell
-# Create a Python virtual environment
-cd ~
-virtualenv venv
-```
-
-To activate your new virtual environment and install `Turi Create` in this environment, follow these steps:
-```shell
-# Active your virtual environment
-source ~/venv/bin/activate
-
-# Install Turi Create in the new virtual environment, pythonenv
-(venv) pip install -U turicreate
-```
-
-Documentation
--------------
-
-The package [User Guide](https://apple.github.io/turicreate/docs/userguide) and [API Docs](https://apple.github.io/turicreate/docs/api) contain
```

Comment:
This section in the PyPI readme should be modified to point to the Github too and we can make it bullet points (see https://github.com/apple/coremltools/blob/master/README.rst#more-information for an example)